monitoring/mock_uss/riddp/routes_observation.py

```diff
@@ -108,7 +109,9 @@ def display_data() -> Tuple[str, int]:
     # Construct detailed flights response
     response = observation_api.GetDisplayDataResponse(flights=[
       _make_flight_observation(f, view) for f in validated_flights])
-    db.flights = flight_info
+    with db as tx:
+        for k, v in flight_info.items():
+            tx[k] = v
```

Comment:
Shouldn't it be tx.flights[k] ?

Reply:
Yes, excellent catch -- we don't have an automated test that would have revealed this bug yet